CMakeLists.txt

```diff
@@ -59,6 +59,31 @@ set(SOURCES
 
 # Generate executable and link
 add_executable(${TARGET} ${SOURCES})
+
```

Comment:
i think it makes sense to to have these definitions in `efm32-base.cmake` rather than here, to separate use of the tools from the supporting logic and because it's common to any `CMakeLists.txt`?

Reply:
I agree, but [`target_link_options`](https://cmake.org/cmake/help/git-stage/command/target_link_options.html) can only be used after we've defined the target via `add_executable`.

Reply:
hmm, fair. what if we put the link options into a cmake function in `efm32-base.cmake` then add a call to `CMakeLists.txt` after `add_executable`? (something like `efm32_configure_linker(${TARGET})`)

Reply:
Something like this?

Reply:
look perfect, thanks ^_^